src/components/networked.js

```diff
@@ -17,6 +17,60 @@ function defaultRequiresUpdate() {
   };
 }
 
+AFRAME.registerSystem("networked", {
+  init() {
+    this.updateNextSyncTime();
+    this.components = [];
+  },
+
+  register(component) {
+    this.components.push(component);
+  },
+
+  deregister(component) {
+    const idx = this.components.indexOf(component);
+
+    if (idx > -1) {
+      this.components.splice(idx, 1);
+    }
+  },
+
+  tick: (function() {
+    const data = { d: [] };
+
+    return function() {
+      if (!NAF.connection.adapter) return;
+      if (!this.needsToSync()) return;
+
+      for (let i = 0, l = this.components.length; i < l; i++) {
+        const c = this.components[i];
+        if (!c.isMine()) continue;
+        if (!c.el.parentElement) continue;
```

Comment:
This should not happen but I think we have seen it. Should at least log an error so we can make sure we see it if it still does happen.